framework/src/org/checkerframework/framework/test/SimpleOptionMap.java

```diff
@@ -0,0 +1,90 @@
+package org.checkerframework.framework.test;
+
+import org.checkerframework.framework.util.PluginUtil;
+
+import java.io.File;
+import java.util.Iterator;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by jburke on 7/7/15.
```

Comment:
Yeah, sorry.  I was just butting up against inference deadlines but I think I can add that now as that was pushed back a bit (but is still a top priority).